ceph_cfg/util_which.py

```diff
@@ -60,6 +60,7 @@ def fget(self):
 which_ceph_mon = memoise_which('ceph-mon')
```

Comment:
Oh just noticed the spelling issue here. I would rather change the spelling from 's' to 'z' on another patch though.